schemas/walletAddresses.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Wallet Address is a way to record additional wallet addresses for an MSA
+ * Should be stored using Itemized and Signature Required
+ * SLIP-0044 Standard: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+ */
+export default {
+  type: "record",
+  name: "Address",
```

Comment:
This seems a little redundant with the "address" field; maybe WalletAddress for the name?

Reply:
I was back and forth. I don't know if WalletAddress was better or if we needed something like ChainAddress (as they all have to be chain addresses?) That said the schema is `WalletAddresses` so I guess the atom would be `WalletAddress`

Open to ideas.

Reply:
Updated to Payment Address with the other payment address changes